simple-chat/package.json

```
@@ -0,0 +1,21 @@
+{
+  "private": true,
+  "scripts": {
+    "start": "webpack-dev-server --mode=development --hot",
+    "build": "webpack --mode=development"
```

Собирать надо с модом продакшен.

```suggestion
    "build": "webpack --mode=production"
```